section3/set10/challengeMoodly/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	args := os.Args[1:]
+
+	// if no user input received, then print usage msg
+	if len(os.Args) != 2 || args[0] == "" {
```

Comment:
The 2nd part of OR (||) can panic, as discussed correct the code such that the 2nd part of OR should be a safe check

Reply:
there are no panics taking place when the code is run